apiservers/portlayer/restapi/configure_port_layer.go

```diff
@@ -90,19 +88,15 @@ func configureAPI(api *operations.PortLayerAPI) http.Handler {
 
 	api.TxtProducer = httpkit.TextProducer()
 
-	netCtx, err := network.NewContext(
-		net.IPNet{
-			IP:   net.IPv4(172, 16, 0, 0),
-			Mask: net.CIDRMask(12, 32),
-		},
-		net.CIDRMask(16, 32),
-		sess)
 	if err != nil {
-		log.Fatalf("failed to create network context: %s", err)
+		log.Fatalf("ExecHandler ERROR: %s", err)
 	}
 
+	handlerCtx := &handlers.HandlerContext{
+		Session: sess,
```

Comment:
FYI - session will go away when I get time to do so, or become global if that's not current feasible. No need to change anything, but so you know the plan.

Session is only really useful when you don't know the morefs of the networks/datastores/clusters/et al that you're working with and those will be globally accessible through the extraconfig object once the PL is migrated to that.